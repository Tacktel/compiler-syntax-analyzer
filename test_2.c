int main(char a, char b, int c) {
    int     i = 1;

    if (1 == 1) {
        i = "";
    }
}