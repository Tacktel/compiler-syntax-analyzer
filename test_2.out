INT:int
WSPACE
ID:main
LPAREN:(
CHAR:char
WSPACE
ID:a
COMMA:,
WSPACE
CHAR:char
WSPACE
ID:b
RPAREN:)
WSPACE
LSCOPE:{
WSPACE
INT:int
WSPACE
ID:i
WSPACE
ASSIGN:=
WSPACE
INTVAL:1
TERM:;
WSPACE
CHAR:char
WSPACE
ID:d
TERM:;
WSPACE
IF:if
WSPACE
LPAREN:(
INTVAL:1
WSPACE
COMP:==
WSPACE
INTVAL:1
RPAREN:)
WSPACE
LSCOPE:{
WSPACE
ID:d
WSPACE
ASSIGN:=
WSPACE
CHARVAL:""
TERM:;
WSPACE
RSCOPE:}
WSPACE
RSCOPE:}
