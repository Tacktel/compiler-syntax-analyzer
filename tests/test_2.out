INT:int
WSPACE
ID:main
LPAREN:(
INT:int
WSPACE
ID:a
COMMA:,
WSPACE
INT:int
WSPACE
ID:b
RPAREN:)
WSPACE
LSCOPE:{
WSPACE
WHILE:while
WSPACE
LPAREN:(
ID:a
WSPACE
COMP:==
WSPACE
INTVAL:2
RPAREN:)
WSPACE
LSCOPE:{
WSPACE
ID:b
WSPACE
ASSIGN:=
WSPACE
ID:b
WSPACE
OP:+
WSPACE
INTVAL:1
TERM:;
WSPACE
RSCOPE:}
WSPACE
RETURN:return
WSPACE
INTVAL:1
TERM:;
WSPACE
RSCOPE:}
